dependencies.props

```diff
@@ -1,10 +1,10 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RyuJITVersion>2.1.0-preview1-26010-03</RyuJITVersion>
+    <RyuJITVersion>2.1.0-preview1-26015-05</RyuJITVersion>
     <ObjectWriterVersion>1.0.19-prerelease-00001</ObjectWriterVersion>
-    <CoreFxVersion>4.5.0-preview1-25925-02</CoreFxVersion>
-    <CoreFxUapVersion>4.6.0-preview1-25925-02</CoreFxUapVersion>
-    <MicrosoftNETCoreNativeVersion>2.1.0-preview1-25925-02</MicrosoftNETCoreNativeVersion>
+    <CoreFxVersion>4.5.0-preview1-26015-06</CoreFxVersion>
+    <CoreFxUapVersion>4.6.0-preview1-26015-06</CoreFxUapVersion>
+    <MicrosoftNETCoreNativeVersion>2.1.0-preview1-26015-06</MicrosoftNETCoreNativeVersion>
```

Comment:
https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Native does not have the -06 version for some reason. -05 should work fine.